Dockerfile.asserts

```diff
@@ -1,26 +1,28 @@
 # Build the manager binary
 FROM golang:1.16 as builder
 
-WORKDIR /go/src/github.com/jaegertracing/jaeger-operator
+WORKDIR /workspace
 
 # Download the dependencies. Doing this, if there are changes in the source
 # code but not in the dependencies to download, the tool to build the image will
 # use the cached dependencies
-COPY go.mod /go/src/github.com/jaegertracing/jaeger-operator/go.mod
-COPY go.sum /go/src/github.com/jaegertracing/jaeger-operator/go.sum
+COPY go.mod go.mod
+COPY go.sum go.sum
 RUN go mod download
 
-COPY . /go/src/github.com/jaegertracing/jaeger-operator/
+COPY tests tests
+
+ENV CGO_ENABLED=0
```

Comment:
As far as i know thats a default value since go 1.16.

Reply:
I don't find anything in the documentation. Could you point to a reference? [I specially checked the Go 1.16 release notes](https://go.dev/doc/go1.16)

Thanks!

Reply:
oh you are right. Mixed it up with [GO111MODULE](https://maelvls.dev/go111module-everywhere/) sorry.